sdcm/cluster.py

```diff
@@ -3754,6 +3755,17 @@ def monitoring_version(self):
     def is_enterprise(self):
         return self.targets["db_cluster"].nodes[0].is_enterprise
 
+    @property
+    def sct_dashboard_json_file(self):
+        if not self._sct_dashboard_json_file:
+            sct_dashboard_json_filename = f"scylla-dash-per-server-nemesis.{self.monitoring_version}.json"
```

Comment:
it is based on Scylla version, not monitoring version

Reply:
fixed

Reply:
my comment was incorrect, I see you reverted it to monitoring_version.
`monitoring_version` name is misleading since it is used for dashboards version, right?